repos/fdbt-site/src/pages/searchOperators.tsx

```diff
@@ -11,82 +13,96 @@ import { getCsrfToken } from '../utils';
+        }
+    };
     return (
-        <div className={`govuk-form-group ${removeOperatorsErrors.length > 0 ? 'govuk-form-group--error' : ''}`}>
-            <fieldset className="govuk-fieldset">
-                <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
-                    <h1 className="govuk-fieldset__heading" id="selected-operators">
-                        Here&apos;s what you have added
-                    </h1>
-                </legend>
-                <fieldset className="govuk-fieldset">
-                    <legend className="govuk-fieldset__legend govuk-fieldset__legend--m">
-                        <h2 className="govuk-fieldset__heading govuk-visually-hidden">Remove selected operators</h2>
-                    </legend>
-                    <div className="govuk-inset-text">
-                        <FormElementWrapper
-                            errors={removeOperatorsErrors}
-                            errorId={removeOperatorsErrorId}
-                            errorClass=""
-                        >
-                            <div className="govuk-checkboxes">
-                                {selectedOperators.map((operator, index) => (
-                                    <div key={operator.nocCode} className="govuk-checkboxes__item">
-                                        <input
-                                            className="govuk-checkboxes__input"
-                                            id={`remove-operator-checkbox-${index}`}
-                                            name="operatorsToRemove"
-                                            value={`${operator.nocCode}#${operator.name}`}
-                                            type="checkbox"
-                                        />
-                                        <label
-                                            className="govuk-label govuk-checkboxes__label"
-                                            htmlFor={`remove-operator-checkbox-${index}`}
-                                        >
-                                            {operator.name} - {operator.nocCode}
-                                        </label>
-                                    </div>
-                                ))}
-                            </div>
-                        </FormElementWrapper>
-                        <input
-                            type="submit"
-                            name="removeOperators"
-                            value={selectedOperators.length > 1 ? 'Remove Operators' : 'Remove Operator'}
-                            id="remove-operators-button"
-                            className="govuk-button govuk-button--secondary govuk-!-margin-top-5"
-                        />
-                    </div>
-                </fieldset>
-            </fieldset>
-        </div>
+        <>
+            <div className={`${removeOperatorsErrors.length > 0 ? 'govuk-form-group--error' : ''}`}>
```

Comment:
![Screenshot 2022-10-28 at 09 41 10](https://user-images.githubusercontent.com/111580926/198544324-a6825030-dc7c-4c38-9711-b2a178f3a87d.png)

Because the button has width 100%, its stretching out to match the extra width this error bar is creating. I think we need to either make the button not stretch out too far, or just remove the red bar.